changelog.txt

```diff
@@ -2,6 +2,64 @@
 
 This is an archive of older changelog entries. Most recent entries are maintained in readme.txt
 
+= Unreleased =
+
+Release Date: Unreleased
+
+* Features:
+	* Added a dashboard under the SEO menu item and moved all persistent notifications to this dashboard.
+	* Added an indicator to the menu and admin bar about pending notifications and SEO problems.
+	* Splits the analysis into content and keyword analysis.
+	* Adds a ton of content feedback about the following properties:
+		* The length of subheadings.
+		* The length of text following a subheading.
+		* The length of paragraphs.
+		* The length of sentences.
+		* The repetition of words at the beginning of sentences.
```

Comment:
Sentence beginnings, complex words, variation of sentence length and length of sentences inside the meta description were postponed.